### ISSUE TITLE
verify_ansible_version fails because pyasn1 too old on RHEL 7.x

### ISSUE THREAD

**Issue by jaryn** [openshift/openshift-ansible#3379]
##### Version
```
[cloud-user@ose3-master-d0h72 ~]$ ansible --version
ansible 2.2.0.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides

atomic-openshift-utils-3.4.44-1.git.0.efa61c6.el7.noarch
openshift-ansible-3.4.44-1.git.0.efa61c6.el7.noarch
python-pyasn1-0.1.6-2.el7.noarch
```

##### Observed Results
```
PLAY [Verify Ansible version is greater than or equal to 2.1.0.0] **************

TASK [Verify Ansible version is greater than or equal to 2.1.0.0] **************
fatal: [localhost]: FAILED! => {
    "failed": true
}

MSG:

The conditional check 'not ansible_version.full | version_compare('2.1.0.0', 'ge')' failed. The error was: pyasn1>=0.1.7

The error appears to have been in '/usr/share/ansible/openshift-ansible/playbooks/common/openshift-cluster/verify_ansible_version.yml': line 8, column 5, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  tasks:
  - name: Verify Ansible version is greater than or equal to 2.1.0.0
    ^ here

```

##### Additional Information
IMHO the dependency on pyasn1 should be either added to some openshift-ansible RPM, or it shouldn't be required.

**Comment by jaryn** [comment]
Another requirement is  `cryptography>=1.1`.
The problem seems to be present on RHEL 7.4 as well

**Comment by jaryn** [comment]
Maybe this is rather an downstream issue. Created BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1422832

**Comment by detiber** [comment]
With the callback plugin, we can probably remove the Ansible version check task. I'm not sure if we'd hit the pyasn issue elsewhere.

I suspect we'd hit the cryptography error regardless.

@sdodson I'm wondering if we need to have Ansible update the RPM dependencies or if they are optional if we should update our spec deps.

**Comment by sdodson** [comment]
That playbook no longer exists in master or release-1.4 branches. Can you please use the latest code and re-open if the problem still exists?

**Comment by sdodson** [comment]
Where do you see the dep on python-cryptography?

**Comment by jaryn** [comment]
Never mind. I fixed this by using RHEL 7.4 __and correct repos__. I forgot to switch to newer repos before.
